nautobot/extras/jobs.py

```diff
@@ -1185,27 +1163,24 @@ def _run_job():
                     if output:
                         job.results["output"] += "\n" + str(output)
 
-                if job.failed:
-                    job.logger.warning("job failed")
-                    job_result.set_status(JobResultStatusChoices.STATUS_FAILED)
-                else:
-                    job.logger.info("job completed successfully")
-                    job_result.set_status(JobResultStatusChoices.STATUS_COMPLETED)
+                job.logger.info("job completed successfully")
 
                 if not commit:
-                    raise AbortTransaction()
+                    raise AbortTransaction("Database changes have been reverted automatically.")
 
         except AbortTransaction:
```

Comment:
I think it would help if we completed #2726 shortly after merging this.

Reply:
I'm thinking that'll just be another one for ripping out `run_job`. Leave out the atomic block while we're at it.